### PR TITLE
Adding explanation to validation errors in lenses

### PR DESCRIPTION
**Pull request by pallavi2209**

### CODE REVIEW

File: api/v1/controllers/lenses.js [hunk 1]
```diff
@@ -39,7 +39,9 @@ function updateLensDetails(seqObj) {
     if (seqObj.hasOwnProperty('sourceName')) {
       seqObj.name = seqObj.sourceName;
     } else {
-      throw new apiErrors.ValidationError();
+      throw new apiErrors.ValidationError({
+        explanation: 'name or source name is required in lens json.',
```
Comment (iamigo) [inline api/v1/controllers/lenses.js]: I think name is always required in lens.json, but sourceName is not a valid option for lens.json. Users should never be posting sourceName--that's for us internally, i.e. if a user overrides the name when posting the lens, then we store the name from lens.json in the "sourceName" column in the db.

Comment (pallavi2209) [inline api/v1/controllers/lenses.js]: Ok, let me check this, add tests and update description.

**Comment by pallavi2209** [comment]
@iamigo updated the PR, but travis failing because of some react module, which is resolved by @annyhe in #328

**Comment by iamigo** [comment]
still waiting on @annyhe review?

**Pull request merged**